src/components/layout-components/page-properties.tsx

```diff
@@ -1,12 +1,289 @@
+        </Typography>
+        {/* TODO: Update with Debounce when backend ready, should this even change? */}
+        <TextField
+          fullWidth
+          multiline
+          placeholder={
+            strings.formatString(
+              strings.editSurveysScreen.editPagesPanel.page,
+              `(${pageNumber})`
+            ) as string
+          }
+          value={surveyPages[pageNumber - 1].title}
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="end">
+                <Edit fontSize="small" color="primary" />
+              </InputAdornment>
+            )
+          }}
+        />
+      </Box>
+      <Box p={2} sx={{ borderBottom: "1px solid #DADCDE" }}>
+        <Typography>{strings.editSurveysScreen.editPagesPanel.title}</Typography>
+        {/* TODO: Update with Debounce when backend ready */}
+        <TextField
+          fullWidth
+          multiline
+          placeholder={strings.editSurveysScreen.editPagesPanel.title}
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="end">
+                <Edit fontSize="small" color="primary" />
+              </InputAdornment>
+            )
+          }}
+        />
+      </Box>
+      {!!pageLayouts.find(layout => layout.id === surveyPages[pageNumber - 1].layoutId && layout.name === Templates.QUESTION) && (
+        <Box p={2} sx={{ borderBottom: "1px solid #DADCDE" }}>
+          <Typography>{strings.editSurveysScreen.editPagesPanel.question}</Typography>
+          {/* TODO: Update with Debounce when backend ready */}
+          <TextField
+            fullWidth
+            label={strings.editSurveysScreen.editPagesPanel.question}
+            size="small"
+            select
+            // TODO: this should handle Multiple question types later
+            defaultValue={QuestionType.SINGLE}
+          >
+            <MenuItem key={QuestionType.SINGLE} value={QuestionType.SINGLE}>
+              {QuestionType.SINGLE}
+            </MenuItem>
+            {/* <MenuItem key={ QuestionType.MULTIPLE } value={ QuestionType.MULTIPLE }>
+{ QuestionType.MULTIPLE }
+</MenuItem> */}
+          </TextField>
+        </Box>
+      )}
+      {!!pageLayouts.find(layout => layout.id === surveyPages[pageNumber - 1].layoutId && layout.name === Templates.QUESTION) &&
+        !!options.length && (
+          <Box p={2} sx={{ borderBottom: "1px solid #DADCDE" }}>
+            {options?.map((option, i) => (
+              renderOptionsWithDebounceTextField(
+                "option",
+                (e) => handleQuestionOptionChange(e, option),
+                option,
+                "option",
+                true,
+                `${i}`
+              )
+            ))}
+          </Box>
+        )}
+      {!!pageLayouts.find(layout => layout.id === surveyPages[pageNumber - 1].layoutId && layout.name === Templates.QUESTION) && (
```

Comment:
This should probably change once the finding of layout mentioned in some comment is implemented.